docs/running-on-kubernetes.md

```diff
@@ -477,6 +477,22 @@ from the other deployment modes. See the [configuration page](configuration.html
   </td>
 </tr>
 <tr>
+  <td><code>spark.kubernetes.executor.labels</code></td>
+  <td>(none)</td>
+  <td>
+    Custom labels that will be added to the executor pods. This should be a comma-separated list of label key-value pairs,
+    where each label is in the format <code>key=value</code>.
```

Comment:
Given that there's precedent I'm fine with not supporting it.  We probably will need to support values containing `=` and `,` at some point in the future though.

Could that be added with `\` escaping or would it take a larger change in the format of `spark.kubernetes.executor.annotations` values?